### DISCUSSION TITLE
The issue of composite modeling

### DISCUSSION THREAD

**Discussion by Gordon198** [neperfepx/neper#866] [Q&A] [answered]
I want to generate a model consisting of a rectangular prism with a cylindrical tunnel embedded in the middle, similar to a tunnel surrounded by rock formations. Then, I would like to generate blocks from the entire model. How can I proceed?

**Comment by rquey** [comment, marked as answer]
You can model one or the other, but not both under the same tessellation / geometry. If your simulation tool allows it, I would advice you to generate one model for each (surrounding rocks and tunnel), and define interactions between the two.
